### PR TITLE
Update documentation on how to host hubs on digital ocean

### PR DESCRIPTION
**Pull request by ulftietze**
Update minor missing information on the tutorial on how to host Hubs on digital ocean.

### CODE REVIEW

**Comment by hrithikwins** [comment]
Hey @ulftietze thanks a lot for adding the missing steps and correcting the command 🖖🏻

File: creator-labs-files/CL-archive/ghost-to-md-output/2024-05-13-hubs-community-edition-how-to-deploy-on-digital-ocean.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ and execute the docker file that we have by running
 
 and later once it's successfully build, generate your file by
 
-> docker run --rm -it -v ./app hubs-ce-builder:latest
+> docker run --rm -it -v .:./app hubs-ce-builder:latest
```
Comment (Exairnous) [inline creator-labs-files/CL-archive/ghost-to-md-output/2024-05-13-hubs-community-edition-how-to-deploy-on-digital-ocean.md]: I believe there should be no `.` after the colon.
```suggestion
> docker run --rm -it -v .:/app hubs-ce-builder:latest
```

**Comment by ulftietze** [comment]
@Exairnous Yes, of course. Thank you very much.

**Comment by Exairnous** [comment]
@ulftietze You're welcome!  Glad to help and thank you for flagging this initially.
@hrithikwins What are your thoughts about merging this?  Remember that it's supposed to be an archive, however, people are coming here to follow the tutorials and we haven't got Creator Labs back up yet.

**Comment by hrithikwins** [comment]
I would highly recommend to get this merged as when we use this archives to get the new one running up quickly this is going to be a good reference

**Comment by Exairnous** [comment]
@hrithikwins This PR only affects the markdown file and not the ghost file that will be used when importing to the new Creator Labs instance.  But I suppose I can also leave a note that this'll have to be updated once the ghost import is done.  I'll run it by the governance team as we haven't defined a merge policy yet.

**Comment by Exairnous** [comment]
@ulftietze Can you please update https://github.com/Hubs-Foundation/hubs-blender-files/blob/main/creator-labs-files/CL-archive/creator-labs.ghost.2024-05-17-15-18-34.json with these changes as well?  After that's done I'll merge this PR.

**Pull request merged**